calico-policies/private-network-isolation/calico-v3/ca-tor/allow-egress-pods-private.yaml

```diff
@@ -0,0 +1,100 @@
+# This policy opens port 53 and 5353 for DNS,
+# port 2049 for communication with NFS file servers,
+# ports 443 and 3260 for communication to block storage,
+# port 10250 for the openVPN pod, port 4443 or 6443 for metrics-server, and
+# port 2040 and 2041 on 172.21.0.0/24 for the master API server local proxy. It also
+# allows pod access to other pods, denies access to private networks but allows access to
+# public networks, and allows access in each zone of the region to the master.
+
+# NOTE: The policies in the ca-tor directory are meant for use with the Toronto multizone location.
+# For the Toronto single zone location, use the policies in the us-east directory instead.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-egress-pods-private
+spec:
+  egress:
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 2049
+      - 3260
+      - 10250
+    protocol: UDP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 4443 # Kubernetes 1.19 or later
+      - 6443 # OpenShift 4.3 or later
+      - 2049
+      - 3260
+      - 10250
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: UDP
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: TCP
+  - action: Allow
+    destination:
+      selector: projectcalico.org/orchestrator == 'k8s'
+    source: {}
+  # Denies pod egress to private networks. If workers are connected to a public VLAN,
+  # pod egress is permitted to public networks.
+  - action: Allow
+    destination:
+      notNets:
+      - 10.0.0.0/8
+      - 172.16.0.0/12
+      - 192.168.0.0/16
+    source: {}
+  - action: Allow
+    destination:
+      nets:
+      # Nets for Toronto multizone location: tor01, tor04, tor05
+      - 166.9.76.6/32
```

Comment:
I think these are for the c103.private.ca-tor.containers.cloud.ibm.com and c104.private.ca-tor.containers.cloud.ibm.com names, which are what the customer kubeconfigs use.  But the workers use the individual DNS names.  So these are the IPs we need to add:

```
Brads-MacBook:~ behle$ dig +short c103-1-1.private.ca-tor.containers.cloud.ibm.com
prod-tor01-tugboat10-535900.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.76.7
Brads-MacBook:~ behle$ dig +short c103-2-1.private.ca-tor.containers.cloud.ibm.com
prod-tor04-tugboat10-913953.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.77.7
Brads-MacBook:~ behle$ dig +short c103-3-1.private.ca-tor.containers.cloud.ibm.com
prod-tor05-tugboat10-950754.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.78.7
Brads-MacBook:~ behle$ dig +short c104-1-1.private.ca-tor.containers.cloud.ibm.com
prod-tor01-tugboat10.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.76.5
Brads-MacBook:~ behle$ dig +short c104-2-1.private.ca-tor.containers.cloud.ibm.com
prod-tor04-tugboat10.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.77.5
Brads-MacBook:~ behle$ dig +short c104-3-1.private.ca-tor.containers.cloud.ibm.com
prod-tor05-tugboat10.ca-tor.serviceendpoint.cloud.ibm.com.
166.9.78.5
```